monitoring/metrics.md

```diff
@@ -0,0 +1,163 @@
+## Requests coming to / of the API
```

Comment:
since we already measure the amount of Github webhooks we receive, this would give us a more granular view into the API as a whole: and transitively into dashboard, since it's the biggest user of the API